client/components/map.jsx

```diff
@@ -0,0 +1,134 @@
+/* global google */
+
+import React from 'react';
+import { GoogleMap, Marker, InfoWindow } from '@react-google-maps/api';
+
+const containerStyle = {
+  width: '100%',
+  height: '100%',
+  position: 'relative'
+};
+
+let center = {
+  lat: 33.680,
+  lng: -117.828
+};
+class MyComponents extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      infoWindow: false,
+      currentLocation: null,
+      places: ''
+    };
+    this.openInfoWindow = this.openInfoWindow.bind(this);
+    this.closeInfoWindow = this.closeInfoWindow.bind(this);
+    this.getLocation = this.getLocation.bind(this);
+    this.createMarker = this.createMarker.bind(this);
+  }
+
+  getLocation() {
+    const map = new google.maps.Map(
+      document.getElementById('map'),
+      {
+        center: center,
+        zoom: 13,
```

Comment:
Instead of querying the DOM, we use a `ref` in React.

```jsx
<div ref={this.mapDivRef}></div>
```

```js
const map = new google.maps.Map(this.mapDivRef.current, {
```

Reply:
ref has been added

![image](https://user-images.githubusercontent.com/48267398/164953940-f48e1037-ee9e-43d6-b430-b82099986bd0.png)
![image](https://user-images.githubusercontent.com/48267398/164953945-50fd8051-87b6-4b44-90aa-bb0cdf616331.png)